lib/config/rules/functionality.js

```diff
@@ -0,0 +1,80 @@
+// Functionality Specific:
+// These rules catch common errors in JS programming or
+// otherwise confusing constructs that are prone to producing bugs:
+// Reference https://palantir.github.io/tslint/rules/
+
+module.exports = {
+  // Bans the use of specific functions or global methods.
+  ban: false,
```

Comment:
Ahhhh, [that's](https://github.com/Shopify/eslint-plugin-shopify/blob/master/lib/config/.eslintrc.js) where it's coming from. Got it. 
Why don't we simply disable it [here](https://github.com/Shopify/eslint-plugin-shopify/blob/master/lib/config/rules/stylistic-issues.js#L125) then?